client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -687,6 +687,75 @@
         };
     }
 
+    CropImageService.$inject = ['urls', '$http'];
+    function CropImageService(urls, $http) {
+        this.saveCrop = function saveCrop(form, item) {
```

Comment:
I would perhaps just pick a different for the `form` parameter, even though it represents form data. Services generally do not not know anything about forms, they operate on data sent to them by other components.